spec/features/transcripts/transcript_preview_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+
+RSpec.feature 'Transcript Preview', type: :feature do
```

Comment:
@RalucaP Not sure if it is fine to remove type: :feature if you're doing `Rspec.feature`.

Reply:
Nice one! Thanks :)